### DISCUSSION TITLE
Discussion for "Microsoft Security Advisory CVE-2023-44487: .NET Denial of Service Vulnerability"

### DISCUSSION THREAD

**Discussion by gfoidl** [dotnet/aspnetcore#51367] [General]
Please use this discussion-thread here for https://github.com/dotnet/aspnetcore/issues/51264

**Comment by gfoidl** [comment]
@amcasey is this is a good idea to steer the discussion to here or not?
It's about
> If a bot comes through and locks it, it should be reopened because this is a discussion forum.

So the issue can remain closed (and potentially locked) once there's a link to here.
We can hide / delete this comment (and your answer to it), so that doesn't disturb.

If it's a bad idea feel free to close this one here.

**Comment by amcasey** [comment]
The original's been reopened, so I don't think we'll need this.  Thanks though!
